### DISCUSSION TITLE
Border around Explorer Patcher in System tray

### DISCUSSION THREAD

**Discussion by Gazdok1** [valinet/ExplorerPatcher#3258] [General]
Hello,
Can anyone please tell how I can get rid of the border / frame around the system tray with EP. I am using Win11.
It is a white frame like border. It also changes the behaviour of the bottom right corner to go back to home page. I uninstalled EP and the border was gone and reinstalled and it was back. Whatever option I change, none seem to get rid of the border.

![border around system tray win11](https://github.com/valinet/ExplorerPatcher/assets/139625309/b4aee0a8-6449-4e0f-bca2-279aba2d47e5)

**Comment by pyrates999** [comment]
to enable or disable: https://answers.microsoft.com/en-us/windows/forum/all/selection-box-on-windows-being-only-a-white-border/4f8deaa3-24ad-4bed-bbca-b26f7f63d4dd

Selection Box on windows (being only a white border)
So ever since i got windows, my selection box has been a white dotted outline. I looked around and the other solutions haven't helped. BUT! i found on that does for those that have the same problem.

Go to the windows search bar
Type "Advanced"
Click on "View advanced system settings"
Click on the performance settings button
Choose the Visual Effects tab, unless already shown.
Untick and then re-tick the "Show translucent selection rectangle" box.
And without a restart your color should be back.

If you want to change the color and dont know how, here is how:
Open your run command or Win + R
Type in "regedit"
HKEY current user ->
Control Panel ->
Colors ->
Hilight and or HotTrackingColor

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2323

Please close this

**Reply by Gazdok1** [reply]
Thanks,

I also found in EP in -Properties -advanced -untick "Enable Advanced Mitigation for correct rendering using classic scheme".

That gets rid of the frame. Not sure why this was ticked.

**Comment by Gazdok1** [comment]
Another comment.
After a recent update - I think - EP is not displaying the internet icon on the taskbar any longer.

**Reply by pyrates999** [reply]
please create a new issue then.

**Comment by Gazdok1** [comment]
Hi,

Do you mean a complete un - and reinstallation?

**Reply by pyrates999** [reply]
No, create a new issue on here.  It will ask you for details that are needed.
